pinner/progress/progress.go

```diff
@@ -0,0 +1,3 @@
+package pinning_progress
```

Comment:
is this worth having as its own package? or own file? seems like declaring a type in pinmgr.go would be ok, or we could have a types.go file in the pinner package to declare this

Reply:
nvm - i see the cycle with cm now